tests/test_meshtastic_backend.py

```
@@ -0,0 +1,294 @@
+            assert backend._callback_registered is True
+            assert backend._message_callback is test_callback
+
+    def test_register_callback_twice(self) -> None:
+        """
+        Verify that registering callback twice skips the second registration.
+        """
+        backend = MeshtasticBackend()
+
+        # First registration
+        with patch("mmrelay.meshtastic_utils.subscribed_to_messages", False):
+            backend.register_message_callback(lambda msg: None)
```

Ruff (0.14.14)

Unused lambda argument: `msg`

(ARG005)